### DISCUSSION TITLE
第六次课程作业(25班)

### DISCUSSION THREAD

**Discussion by zcg-joker** [InternLM/Tutorial#476] [作业]
作业放在这里。
笔记**不放在**这里来。

**Comment by FableFatale** [comment]
https://www.cnblogs.com/yunyetui/articles/17981685

**Comment by zengya55** [comment]
https://github.com/zengya55/InterLM_Learning/blob/main/lesson6/lesson6%20opencompass%E4%BD%9C%E4%B8%9A.md

**Comment by BaiYu96** [comment]
https://github.com/BaiYu96/internLM-learm/blob/main/homework/lesson-6.md

**Comment by liliang1918** [comment]
https://blog.csdn.net/lalala12ll/article/details/135856497?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135856497%22%2C%22source%22%3A%22lalala12ll%22%7D

**Comment by ykuang** [comment]
https://github.com/ykuang/internlm-tutorial/blob/main/notes/lesson6_homework.md

**Comment by sxswz213** [comment]
https://blog.csdn.net/sxswz/article/details/135886769?spm=1001.2014.3001.5501

**Comment by Cyberworldvoyager** [comment]
https://github.com/Cyberworldvoyager/Interlm_lecture/blob/main/lecture%206.pdf

**Comment by wade361** [comment]
https://github.com/wade361/-Wade/issues/10

**Comment by GuoYiFantastic** [comment]
https://github.com/GuoYiFantastic/InternLM_training_camp/blob/main/OpenCompass%20%E5%A4%A7%E6%A8%A1%E5%9E%8B%E8%AF%84%E6%B5%8B/%E4%BD%9C%E4%B8%9A.md
